.circleci/config.yml

```diff
@@ -58,4 +58,4 @@ workflows:
             branches:
               only: /.*/
             tags:
-              only: /^[0-9].*-r[0-9]+$/
+              only: /^v[0-9].*+$/
```

Comment:
We expect release tags to have the `v` prefix as the Kubeless release tags also have this suffix.